Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM yastdevel/cpp
+RUN zypper --gpg-auto-import-keys --non-interactive in --no-recommends \
+  "pkgconfig(gio-unix-2.0)" \
+  "pkgconfig(glib-2.0) >= 2.31.0" \
+  "pkgconfig(gtk+-3.0)" \
+  "pkgconfig(libgnome-menu-3.0)"
```

Comment:
I just didn't know the docker image build process in depth. I thought that docker builds a kind of "chroot" and packs it into an image and solves dependencies when building. If list of packages has to be provided by user then I'm fine with this approach.